bundles/binding/org.openhab.binding.panstamp/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,31 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.binding.panstamp.internal
+Ignore-Package: org.openhab.binding.panstamp.internal
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Name: openHAB PanStamp Binding
+Bundle-SymbolicName: org.openhab.binding.panstamp
+Bundle-Vendor: openHAB.org
+Bundle-Version: 1.8.0.qualifier
+Bundle-ManifestVersion: 2
+Bundle-Description: This is the PanStamp binding of the open Home Aut
+ omation Bus (openHAB)
+Import-Package: org.apache.commons.lang,
+ org.openhab.core.binding,
+ org.openhab.core.events,
+ org.openhab.core.items,
+ org.openhab.core.library.items,
+ org.openhab.core.library.types,
+ org.openhab.core.types,
+ org.openhab.model.item.binding,
+ org.osgi.framework,
+ org.osgi.service.component,
+ org.osgi.service.event,
+ org.slf4j
+Export-Package: org.openhab.binding.panstamp
+Bundle-DocURL: http://www.openhab.org
+Bundle-RequiredExecutionEnvironment: J2SE-1.5
+Service-Component: OSGI-INF/binding.xml, OSGI-INF/genericbindingprovider.xml
+Bundle-ClassPath: .,
```

Comment:
While not strictly required for OH1, could you add an about.html similiar to https://github.com/openhab/openhab2/blob/master/addons/binding/org.openhab.binding.avmfritz/about.html showing the additional libraries you are using and the licenses they are distributed with?